linux_os/guide/system/accounts/accounts-physical/bootloader/bootloader_password.rule

```diff
@@ -2,7 +2,7 @@ documentation_complete: true
 
 prodtype: rhel7,fedora
 
-title: 'Set Boot Loader Password'
+title: 'Set Boot Loader Password in grub2 Configuration'
```

Comment:
We really don't need to add `in grub2 Configuration` here which would make rhel6 integration easier.

Reply:
Ultimately this is for GRUB2 which is significantly different from GRUB legacy. The "bootloader" folder should probably be structured into a `grub-legacy` and `grub2` folder, with rules having different titles and different file names to distinguish the two. But those changes are out of scope for this PR.

Reply:
Hmmm..... good point.

Reply:
@cipherboy maybe instead of grub-legacy, make it grub.cfg? I just realized that people who may not have made the switch to RHEL6 won't understand the connotation here.

Reply:
@cipherboy was about to merge to....

Reply:
@redhatrises "Made the switch to RHEL6?" -- I think you meant to comment on the RHEL 6 file?... I updated that one to "grub.conf" since that's the name of the file in RHEL 6. This is against RHEL 7 and it says "in grub2" in the updated version (as GH shows the outdated source) -- that's the official name of the package. `grub-legacy` has ~5M hits on Google too, so I'd definitely consider it a valid, easily discoverable name that people should be familiar with anyways.